src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let res = 0;
+
+  for (let i = 0; i < days; i++) {
+    res += 40;
+  }
+
+  if (days >= 3 && days < 7) {
+    res -= 20;
+  }
+
+  if (days >= 7) {
+    res -= 50;
```

Comment:
use constants for values ​​that never change.
for example:
```suggestion
    return basePrise - LONG_TERM_DISCOUNT;
```